v3/doio/kb30/kb30-01.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "DOIO",
+  "vendorId": "0xD010",
+  "productId": "0x3001",
+  "keycodes": ["qmk_lighting"],
+  "menus": ["qmk_rgb_matrix"],
```

Comment:
https://github.com/qmk/qmk_firmware/pull/19395/files#diff-47d8ce7c695a8cadbba1922ca5ff8649ec5e62eec7b69c3e355635560df66f25R77-R120

Your default configuration does **not** enable all the RGB Matrix effects.

However the VIA default configuration does.

The user will experience the name of the effect does not matchup to the effect displayed on the board.
Please look at 
https://github.com/the-via/reader/blob/master/src/common-menus/qmk_rgb_matrix.ts

https://www.caniusevia.com/docs/built_in_menus

An example of another keyboard without all the animations defined is 
https://github.com/the-via/keyboards/blob/master/v3/horrortroll/handwired_k552/handwired_k552.json

Reply:
@Xelus22  Hi,thank you for your advice,I've changed the file.